workers/docker-worker/package.json

```diff
@@ -80,7 +80,6 @@
     "ws": "^7.1.2"
   },
   "devDependencies": {
-    "github": "^13.1.0",
```

Comment:
Why was this removed?

Reply:
It's unused -- never require'd.  And it's an ancient version, which was pulling in old deps.